ftdetect/markdown.vim

```diff
@@ -1,3 +1,3 @@
 " markdown filetype file
-au BufRead,BufNewFile *.{md,mdown,mkd,mkdn,markdown,mdwn}   set filetype=mkd.markdown
```

Comment:
@yuex sorry, I don't understand very well what will be the problem with `mkd.markdown`. Can you give a minimal example? What do you mean by: "reserve those mkd.xxx with those new markdown.xxx files". Did you mean `preserve`?